lib/land/trackers/api_tracker.rb

```diff
@@ -103,6 +103,10 @@ def user_agent
         @user_agent = UserAgent[user_agent]
       end
 
+      def raw_user_agent
+        request.params['user_agent'] || user_agent.user_agent || Land.config.blank_user_agent_string
```

Comment:
Here, we could probably actually remove the last check returning Land.config.blank_user_agent_string, since that's also used in the `#user_agent` method to _always_ return a UserAgent object with that default string set.

Reply:
We only ever want to send the user agent from params or none, as the user agent in the header from NextJS will be wrong, always `NextJS ....(something)`

```suggestion
        request.params['user_agent'] ||  Land.config.blank_user_agent_string
```